velox/docs/functions/presto/map.rst

```diff
@@ -96,7 +96,8 @@ Map Functions
 
 .. function:: map_top_n(map(K,V), n) -> map(K, V)
 
-    Truncates map items. Keeps only the top N elements by value.
+    Truncates map items. Keeps only the top N elements by value. Keys are used to break ties on non-NULL values. Both keys and values should be orderable.
```

Comment:
> Keys are used to break ties on non-NULL values.

Could you elaborate how keys are used to break ties? E.g., when there are multiple map entries with the same value, the entry with max/min key is taken.